build.go

```diff
@@ -50,13 +50,13 @@ func compressionCodecString(c sequencefile.CompressionCodec) string {
 	}
 }
 
-func (vs *version) build() {
+func (vs *version) build(rebuild bool) {
```

Comment:
Total nit, and I don’t know about the idiomacy of this in golang, but I dislike Boolean flags like this preferring an explicit second function with reuse.

Reply:
Yeah I think you're right here, I'll update with reuse.